src/managers/GuildManager.js

```diff
@@ -129,6 +130,8 @@ class GuildManager extends BaseManager {
    * <warn>This is only available to bots in fewer than 10 guilds.</warn>
    * @param {string} name The name of the guild
    * @param {Object} [options] Options for the creating
+   * @param {number} [options.afkChannelID] The ID of the AFK channel
```

Comment:
```suggestion
   * @param {Snowflake} [options.afkChannelID] The ID of the AFK channel
```

Reply:
This isn't actually wrong per say, as both Snowflakes as int or strings work in the create guild endpoint 👀